src/App.tsx

```diff
@@ -1,20 +1,19 @@
-import { PeoplePage } from './components/PeoplePage';
-import { Navbar } from './components/Navbar';
-
 import './App.scss';
+import { Outlet } from 'react-router-dom';
+import { Navbar } from './components/Navbar';
 
 export const App = () => {
   return (
     <div data-cy="app">
+      <html className="has-navbar-fixed-top"></html>
```

Comment:
Using the `<html>` tag inside a React component's JSX is not appropriate. The `<html>` tag is meant to be used at the document level, not within components. Consider removing this line or replacing it with a more suitable HTML element, such as a `<div>` or `<header>`, depending on your layout needs.